packages/patternfly-4/src/templates/contentTemplate.js

```diff
@@ -20,7 +20,7 @@ import CoreSideNav from '../components/_core/Documentation/SideNav';
 import './template.scss';
 
 const components = {
-  a: props => <a {...props} href={props.href ? props.href.toLowerCase() : props.href} />
+  a: props => <a {...props} href={props.href} />
```

Comment:
you could remove this component now since we're not doing anything special to it